web-registry/src/components/organisms/ProjectStorySection/ProjectStorySection.tsx

```diff
@@ -0,0 +1,101 @@
+import ReactPlayerLazy from 'react-player/lazy';
+import { Box, Grid } from '@mui/material';
+import cx from 'clsx';
+
+import ReadMore from 'web-components/lib/components/read-more';
+import Section from 'web-components/lib/components/section';
+import { Label, Title } from 'web-components/lib/components/typography';
+
+import { useProjectStorySectionStyles } from './ProjectStorySection.styles';
+import { ProjectStorySectionProps } from './ProjectStorySection.types';
+import { parseProjectPageMetadata } from './ProjectStorySection.utils';
+
+export function ProjectStorySection({
+  projectPageMetadata,
+}: ProjectStorySectionProps): JSX.Element | null {
+  const { storyMedia, storyTitle, story } =
+    parseProjectPageMetadata(projectPageMetadata);
+  const { classes } = useProjectStorySectionStyles();
+
+  return storyMedia || (storyTitle && story) ? (
+    <Section
+      sx={{
+        root: {
+          overflow: { md: 'visible' },
+          pb: { xs: 19.75, sm: 25 },
+          ...(storyTitle &&
+            story &&
+            storyMedia && { pr: { tablet: 2.5, lg: 0 } }),
+        },
+      }}
+    >
+      <Grid container>
+        {storyTitle && story && (
```

Comment:
Optional suggestion, having 2 child components here for readability:
`<StoryTitle />` and `<StoryMedia />`